<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metodosnumericos</class>
 <widget class="QMainWindow" name="metodosnumericos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>metodosnumericos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Metodos Numericos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="diferenciacion">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Diferenciacion
Numerica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gauss">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Eliminacion
 Gaussiana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="euler1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo de Euler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regresionmultiple">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Regresion
Multiple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="biseccion1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Biseccion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lagrange1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolacion
de lagrange</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="regresionlineal">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Regresion
Lineal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="puntofijo1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Punto Fijo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seidel1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauss-Seidel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="integracion">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Integracion
Numerica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newton1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Newton
Raphson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kutta1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Runge-Kutta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
